ui/apps/platform/src/Containers/Vulnerabilities/VirtualMachineCves/VirtualMachine/VirtualMachineVulnerabilitiesTable.tsx

```diff
@@ -0,0 +1,114 @@
+import VirtualMachineComponentsTable from './VirtualMachineComponentsTable';
+
+export type VirtualMachineVulnerabilitiesTableProps = {
+    tableState: TableUIState<CveTableRow>;
+};
+
+function VirtualMachineVulnerabilitiesTable({
+    tableState,
+}: VirtualMachineVulnerabilitiesTableProps) {
+    const COL_SPAN = 7;
+    const expandedRowSet = useSet<string>();
+
+    return (
+        <Table
+            borders={tableState.type === 'COMPLETE'}
+            variant="compact"
+            aria-live="polite"
+            aria-busy={false}
+        >
+            <Thead>
+                <Tr>
+                    <ExpandRowTh />
+                    <Th>CVE</Th>
+                    <Th>Severity</Th>
+                    <Th>CVE status</Th>
+                    <Th>CVSS</Th>
+                    <Th>EPSS probability</Th>
+                    <Th>Affected components</Th>
+                </Tr>
+            </Thead>
+            <TbodyUnified
+                tableState={tableState}
+                colSpan={7}
+                errorProps={{
+                    title: 'There was an error loading results',
+                }}
+                emptyProps={{
+                    message: 'No CVEs were detected for this virtual machine',
+                }}
+                renderer={({ data }) =>
+                    data.map((vulnerability, rowIndex) => {
+                        const isExpanded = expandedRowSet.has(vulnerability.cve);
+                        return (
+                            <Tbody key={vulnerability.cve} isExpanded={isExpanded}>
+                                <Tr>
+                                    <Td
+                                        expand={{
+                                            rowIndex,
+                                            isExpanded,
+                                            onToggle: () =>
+                                                expandedRowSet.toggle(vulnerability.cve),
+                                        }}
+                                    />
+                                    <Td dataLabel="CVE">{vulnerability.cve} </Td>
+                                    <Td dataLabel="Severity">
+                                        <VulnerabilitySeverityIconText
+                                            severity={vulnerability.severity}
+                                        />
+                                    </Td>
+                                    <Td dataLabel="CVE status">
+                                        <VulnerabilityFixableIconText
+                                            isFixable={vulnerability.isFixable}
+                                        />
+                                    </Td>
+                                    <Td dataLabel="CVSS">
+                                        <CvssFormatted
+                                            cvss={vulnerability.cvss}
+                                            scoreVersion="v3"
```

Comment:
Do we need to double check with scanner team about CVSS version number?

Reply:
Yes definitely. I added this as a topic of conversation for our sync tomorrow.